### PR TITLE
Create NYT_Style.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/NYT_Style.com.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
+	For other New York Times coverage, see NYTimes.xml.
+
+	Not exist:
+		^
+		www
+
+	Timeout by travis:
+		m.cn.nytstyle.com
```
Comment (J0WI) [inline src/chrome/content/rules/NYT_Style.com.xml]: Maybe a custom test url works?

**Comment by J0WI** [comment]
I have some mixed content issues. Can you confirm that this breaks nothing?

**Comment by ivysrono** [comment]
@J0WI Where are the mixedcontent issues？

**Comment by J0WI** [comment]
on $ from graphics8.nytimes.com and js.nyt.com

**Pull request merged**